.github/workflows/ci.yaml

```diff
@@ -0,0 +1,31 @@
+name: scala-lang docs
+
+on:
+  pull_request:
+  schedule:
+    - cron: '00 0 * * *'
```

Comment:
@SethTisue 
This schedule should be updated per your preference.

Reply:
I am not sure why we need this? (there seems to be something similar in the Travis CI settings, scheduling a build every day unless there was already an automatic build triggered by a PR)
The only thing I can think about is that this would allow us to detect broken external links sooner?

Reply:
I think schedule is needed because validation logic is different for pull requests and scheduled run

Reply:
> this would allow us to detect broken external links sooner

I believe that's the motivation, yes.